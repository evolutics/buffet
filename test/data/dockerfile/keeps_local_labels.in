FROM alpine AS example
ARG example
LABEL org.opencontainers.image.title="Example"

FROM alpine
ARG example
