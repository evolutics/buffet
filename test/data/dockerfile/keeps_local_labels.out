ARG example=''

ARG _alpine_version='3.9.4'

FROM alpine AS example
ARG example
LABEL org.opencontainers.image.title="Example"

FROM alpine:"${_alpine_version}"

ARG example

WORKDIR /workdir
